metadata/src/main/java/edu/unc/lib/dl/rdf/Cdr.java

```diff
@@ -95,7 +95,7 @@ public static String getURI() {
             "http://cdr.unc.edu/definitions/model#invalidTerm" );
 
     /** Relation from an AggregateWork to a child which will be treated as the primary
-     *  work for this object.
+     *  object for this work.
```

Comment:
The ol' switch-a-roo

Reply:
Might as well change the documentation about it from "AggregateWork" to just "Work" while you're at it.

Reply:
👍